mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_001/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_001.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_001//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_001//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_001//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_001//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_001//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_001//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_001//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_001//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_001//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_002/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_002.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_002//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_002//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_002//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_002//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_002//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_002//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_002//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_002//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_002//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_003/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_003.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_003//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_003//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_003//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_003//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_003//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_003//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_003//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_003//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_003//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_004/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_004.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_004//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_004//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_004//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_004//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_004//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_004//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_004//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_004//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_004//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_005/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_005.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_005//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_005//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_005//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_005//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_005//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_005//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_005//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_005//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_005//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_006/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_006.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_006//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_006//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_006//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_006//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_006//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_006//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_006//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_006//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_006//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_007/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_007.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_007//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_007//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_007//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_007//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_007//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_007//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_007//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_007//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_007//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_008/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_008.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_008//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_008//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_008//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_008//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_008//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_008//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_008//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_008//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_008//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_009/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_009.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_009//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_009//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_009//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_009//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_009//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_009//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_009//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_009//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_009//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_010/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_010.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_010//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_010//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_010//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_010//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_010//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_010//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_010//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_010//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_010//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_011/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_011.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_011//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_011//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_011//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_011//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_011//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_011//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_011//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_011//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_011//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_012/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_012.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_012//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_012//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_012//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_012//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_012//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_012//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_012//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_012//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_012//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_013/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_013.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_013//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_013//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_013//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_013//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_013//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_013//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_013//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_013//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_013//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_014/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_014.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_014//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_014//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_014//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_014//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_014//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_014//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_014//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_014//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_014//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_015/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_015.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_015//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_015//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_015//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_015//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_015//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_015//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_015//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_015//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_015//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_016/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_016.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_016//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_016//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_016//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_016//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_016//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_016//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_016//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_016//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_016//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_017/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_017.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_017//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_017//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_017//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_017//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_017//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_017//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_017//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_017//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_017//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_018/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_018.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_018//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_018//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_018//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_018//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_018//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_018//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_018//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_018//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_018//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_019/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_019.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_019//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_019//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_019//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_019//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_019//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_019//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_019//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_019//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_019//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_020/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_020.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_020//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_020//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_020//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_020//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_020//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_020//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_020//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_020//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_020//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_021/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_021.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_021//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_021//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_021//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_021//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_021//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_021//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_021//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_021//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_021//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_022/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_022.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_022//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_022//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_022//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_022//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_022//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_022//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_022//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_022//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_022//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_023/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_023.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_023//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_023//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_023//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_023//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_023//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_023//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_023//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_023//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_023//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_024/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_024.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_024//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_024//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_024//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_024//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_024//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_024//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_024//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_024//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_024//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_025/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_025.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_025//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_025//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_025//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_025//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_025//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_025//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_025//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_025//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_025//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_026/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_026.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_026//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_026//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_026//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_026//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_026//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_026//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_026//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_026//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_026//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_027/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_027.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_027//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_027//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_027//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_027//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_027//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_027//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_027//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_027//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_027//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_028/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_028.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_028//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_028//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_028//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_028//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_028//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_028//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_028//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_028//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_028//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_029/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_029.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_029//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_029//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_029//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_029//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_029//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_029//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_029//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_029//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_029//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_030/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_030.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_030//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_030//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_030//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_030//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_030//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_030//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_030//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_030//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_030//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_031/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_031.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_031//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_031//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_031//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_031//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_031//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_031//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_031//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_031//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_031//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_032/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_032.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_032//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_032//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_032//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_032//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_032//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_032//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_032//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_032//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_032//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_033/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_033.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_033//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_033//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_033//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_033//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_033//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_033//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_033//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_033//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_033//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_034/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_034.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_034//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_034//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_034//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_034//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_034//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_034//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_034//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_034//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_034//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_035/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_035.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_035//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_035//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_035//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_035//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_035//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_035//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_035//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_035//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_035//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_036/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_036.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_036//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_036//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_036//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_036//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_036//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_036//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_036//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_036//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_036//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_037/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_037.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_037//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_037//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_037//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_037//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_037//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_037//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_037//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_037//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_037//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_038/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_038.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_038//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_038//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_038//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_038//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_038//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_038//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_038//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_038//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_038//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_039/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_039.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_039//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_039//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_039//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_039//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_039//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_039//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_039//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_039//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_039//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_040/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_040.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_040//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_040//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_040//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_040//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_040//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_040//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_040//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_040//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_040//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_041/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_041.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_041//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_041//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_041//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_041//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_041//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_041//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_041//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_041//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_041//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_042/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_042.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_042//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_042//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_042//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_042//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_042//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_042//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_042//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_042//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_042//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_043/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_043.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_043//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_043//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_043//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_043//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_043//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_043//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_043//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_043//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_043//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_044/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_044.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_044//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_044//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_044//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_044//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_044//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_044//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_044//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_044//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_044//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_045/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_045.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_045//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_045//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_045//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_045//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_045//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_045//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_045//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_045//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_045//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_046/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_046.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_046//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_046//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_046//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_046//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_046//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_046//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_046//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_046//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_046//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_047/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_047.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_047//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_047//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_047//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_047//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_047//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_047//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_047//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_047//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_047//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_048/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_048.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_048//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_048//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_048//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_048//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_048//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_048//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_048//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_048//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_048//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_049/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_049.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_049//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_049//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_049//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_049//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_049//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_049//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_049//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_049//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_049//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_050/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_050.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_050//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_050//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_050//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_050//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_050//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_050//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_050//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_050//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_050//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_051/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_051.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_051//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_051//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_051//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_051//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_051//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_051//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_051//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_051//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_051//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_052/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_052.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_052//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_052//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_052//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_052//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_052//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_052//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_052//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_052//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_052//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_053/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_053.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_053//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_053//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_053//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_053//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_053//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_053//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_053//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_053//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_053//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_054/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_054.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_054//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_054//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_054//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_054//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_054//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_054//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_054//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_054//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_054//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_055/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_055.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_055//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_055//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_055//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_055//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_055//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_055//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_055//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_055//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_055//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_056/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_056.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_056//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_056//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_056//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_056//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_056//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_056//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_056//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_056//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_056//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_057/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_057.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_057//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_057//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_057//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_057//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_057//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_057//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_057//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_057//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_057//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_058/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_058.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_058//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_058//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_058//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_058//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_058//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_058//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_058//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_058//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_058//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_059/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_059.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_059//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_059//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_059//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_059//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_059//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_059//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_059//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_059//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_059//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_060/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_060.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_060//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_060//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_060//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_060//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_060//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_060//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_060//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_060//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_060//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_061/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_061.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_061//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_061//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_061//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_061//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_061//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_061//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_061//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_061//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_061//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_062/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_062.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_062//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_062//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_062//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_062//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_062//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_062//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_062//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_062//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_062//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_063/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_063.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_063//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_063//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_063//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_063//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_063//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_063//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_063//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_063//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_063//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_064/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_064.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_064//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_064//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_064//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_064//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_064//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_064//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_064//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_064//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_064//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_065/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_065.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_065//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_065//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_065//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_065//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_065//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_065//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_065//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_065//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_065//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_066/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_066.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_066//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_066//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_066//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_066//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_066//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_066//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_066//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_066//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_066//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_067/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_067.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_067//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_067//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_067//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_067//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_067//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_067//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_067//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_067//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_067//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_068/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_068.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_068//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_068//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_068//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_068//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_068//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_068//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_068//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_068//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_068//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_069/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_069.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_069//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_069//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_069//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_069//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_069//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_069//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_069//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_069//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_069//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_070/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_070.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_070//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_070//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_070//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_070//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_070//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_070//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_070//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_070//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_070//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_071/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_071.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_071//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_071//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_071//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_071//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_071//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_071//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_071//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_071//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_071//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_072/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_072.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_072//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_072//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_072//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_072//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_072//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_072//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_072//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_072//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_072//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_073/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_073.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_073//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_073//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_073//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_073//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_073//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_073//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_073//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_073//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_073//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_074/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_074.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_074//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_074//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_074//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_074//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_074//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_074//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_074//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_074//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_074//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_075/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_075.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_075//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_075//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_075//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_075//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_075//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_075//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_075//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_075//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_075//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_076/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_076.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_076//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_076//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_076//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_076//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_076//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_076//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_076//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_076//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_076//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_077/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_077.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_077//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_077//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_077//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_077//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_077//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_077//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_077//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_077//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_077//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_078/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_078.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_078//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_078//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_078//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_078//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_078//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_078//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_078//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_078//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_078//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_079/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_079.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_079//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_079//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_079//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_079//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_079//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_079//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_079//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_079//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_079//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_080/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_080.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_080//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_080//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_080//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_080//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_080//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_080//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_080//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_080//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_080//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_081/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_081.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_081//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_081//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_081//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_081//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_081//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_081//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_081//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_081//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_081//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_082/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_082.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_082//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_082//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_082//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_082//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_082//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_082//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_082//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_082//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_082//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_083/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_083.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_083//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_083//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_083//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_083//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_083//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_083//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_083//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_083//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_083//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_084/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_084.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_084//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_084//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_084//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_084//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_084//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_084//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_084//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_084//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_084//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_085/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_085.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_085//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_085//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_085//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_085//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_085//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_085//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_085//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_085//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_085//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_086/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_086.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_086//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_086//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_086//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_086//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_086//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_086//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_086//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_086//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_086//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_087/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_087.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_087//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_087//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_087//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_087//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_087//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_087//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_087//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_087//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_087//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_088/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_088.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_088//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_088//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_088//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_088//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_088//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_088//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_088//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_088//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_088//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_089/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_089.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_089//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_089//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_089//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_089//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_089//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_089//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_089//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_089//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_089//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_090/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_090.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_090//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_090//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_090//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_090//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_090//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_090//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_090//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_090//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_090//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_091/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_091.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_091//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_091//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_091//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_091//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_091//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_091//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_091//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_091//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_091//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_092/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_092.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_092//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_092//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_092//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_092//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_092//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_092//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_092//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_092//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_092//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_093/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_093.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_093//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_093//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_093//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_093//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_093//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_093//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_093//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_093//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_093//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_094/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_094.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_094//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_094//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_094//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_094//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_094//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_094//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_094//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_094//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_094//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_095/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_095.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_095//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_095//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_095//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_095//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_095//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_095//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_095//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_095//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_095//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_096/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_096.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_096//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_096//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_096//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_096//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_096//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_096//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_096//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_096//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_096//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_097/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_097.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_097//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_097//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_097//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_097//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_097//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_097//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_097//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_097//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_097//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_098/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_098.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_098//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_098//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_098//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_098//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_098//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_098//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_098//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_098//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_098//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_099/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_099.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_099//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_099//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_099//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_099//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_099//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_099//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_099//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_099//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_099//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_100/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_100.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_100//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_100//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_100//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_100//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_100//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_100//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_100//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_100//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_100//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_101/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_101.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_101//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_101//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_101//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_101//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_101//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_101//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_101//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_101//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_101//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_102/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_102.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_102//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_102//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_102//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_102//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_102//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_102//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_102//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_102//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_102//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_103/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_103.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_103//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_103//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_103//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_103//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_103//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_103//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_103//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_103//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_103//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_104/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_104.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_104//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_104//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_104//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_104//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_104//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_104//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_104//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_104//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_104//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_105/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_105.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_105//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_105//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_105//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_105//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_105//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_105//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_105//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_105//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_105//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_106/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_106.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_106//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_106//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_106//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_106//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_106//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_106//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_106//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_106//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_106//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_107/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_107.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_107//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_107//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_107//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_107//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_107//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_107//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_107//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_107//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_107//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_108/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_108.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_108//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_108//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_108//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_108//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_108//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_108//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_108//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_108//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_108//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_109/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_109.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_109//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_109//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_109//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_109//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_109//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_109//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_109//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_109//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_109//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_110/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_110.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_110//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_110//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_110//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_110//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_110//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_110//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_110//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_110//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_110//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_111/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_111.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_111//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_111//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_111//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_111//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_111//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_111//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_111//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_111//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_111//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_112/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_112.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_112//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_112//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_112//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_112//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_112//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_112//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_112//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_112//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_112//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_113/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_113.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_113//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_113//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_113//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_113//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_113//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_113//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_113//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_113//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_113//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_114/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_114.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_114//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_114//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_114//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_114//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_114//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_114//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_114//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_114//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_114//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_115/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_115.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_115//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_115//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_115//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_115//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_115//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_115//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_115//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_115//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_115//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_116/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_116.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_116//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_116//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_116//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_116//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_116//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_116//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_116//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_116//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_116//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_117/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_117.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_117//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_117//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_117//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_117//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_117//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_117//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_117//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_117//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_117//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_118/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_118.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_118//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_118//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_118//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_118//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_118//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_118//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_118//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_118//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_118//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_119/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_119.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_119//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_119//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_119//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_119//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_119//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_119//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_119//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_119//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_119//input.count.txt
mkdir -p /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_120/ && cutadapt --quiet -m 20 -O 20 -a "polyA=A{20}" -a "QUALITY=G{20}" -n 2 /home/gwiktorin/tests/SARS-CoV-2//idemuxxed/sample_120.fastq.gz | cutadapt --quiet -m 20 -O 3 --nextseq-trim=10 -a "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=3;max_error_rate=0.100000" - | cutadapt --quiet -m 20 -O 3 -a "r1polyA=A{18}" - | cutadapt --quiet -m 20 -O 20 -g "r1adapter=AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC;min_overlap=20" --discard-trimmed -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_120//R1.fastq.gz - && bowtie2 -x /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/sequences -L 10 --mm -U /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_120//R1.fastq.gz --quiet -S /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_120//Aligned.out.sam && featureCounts -a /home/gwiktorin/git/sars-cov-2_analysis/targets_SIRV109/annotation.gtf -o /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_120//src.byfeature -t feature -g target_id -M --fraction -p -B -d 30 -s 2 -C /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_120//Aligned.out.sam 2> /dev/null && samtools view -f 4 /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_120//Aligned.out.sam | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_120//unmapped.count.txt && zcat /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_120//R1.fastq.gz | grep -e "^@" | wc -l > /home/gwiktorin/tests/SARS-CoV-2//analysis/sample_120//input.count.txt
